src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/bubbleVisualization.js

```diff
@@ -1,38 +1,42 @@
 class BubbleVisualization extends Visualization {
```

Comment:
Is this visualization really needed? Wasn't it just some kind of a nice demo 4 months ago using dummy code? Should it exist now? The same about ensogl-one.

Reply:
No, these are perfect examples to users. They should not be enabled in GUI though.

Reply:
So this vis can simply not be registered, and still stay in this place.